modules/ROOT/pages/for-each-scope-concept.adoc

```diff
@@ -95,6 +88,15 @@ In case the message inside the For Each scope is persisted, not only the item in
 To avoid this issue you must first remove the rootMessage variable from the message before persisting it.
 ////
 
+=== Differences between For Each and Split Aggregate Scopes
+
+Both the For Each and the xref:split-aggregate-scope.adoc[Split Aggregate scope] split the defined collection and process each element in it with the components within the scope. Also, in both cases, each route runs with the same initial context.
+The difference between these two scopes are:
+
+* For Each works sequentially, while the Split Aggregate works in parallel.
+* For Each does not modify the payload, while the Split Aggregate outputs a collection of the output messages of each iteration.
+* Error handling: Because of the first-mentioned difference, For Each execution gets interrupted when an error is raised (and the Error Handler will be invoked), while Split Aggregate processes each route and then invoke the xref:intro-error-handlers.adoc[Error Handler] with the aggregation of the results and errors of all the routes.
```

Comment:
```suggestion
* Error handling: Because of the first-mentioned difference, For Each execution gets interrupted when an error is raised (and the Error Handler will be invoked), while Split Aggregate processes each route and then invokes the xref:intro-error-handlers.adoc[Error Handler] with the aggregation of the results and errors of all the routes.
```